vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -30,6 +31,18 @@ def get_services(inventory)
 
     def get_pods(inventory)
       process_collection(inventory["pod"], :container_groups) { |n| parse_pod(n) }
+      @data[:container_groups].each do |x|
```

Comment:
I prefer "x" because in this code that is generally copy/pasted you'll soon find "cg" all over the code :smile:. Solution would be to make a generic function out of these and we would have to come up with a generic index anyway.

Anyway I'll change it as you suggest.